api/forms.py

```diff
@@ -32,8 +32,17 @@ class CreatePropertyForm(forms.Form):
 	startPrice = forms.DecimalField(initial=600)
 	dateStart = forms.DateTimeField(initial=datetime.now())
 	dateEnd = forms.DateTimeField(initial=datetime.now())
```

Comment:
Also, right now, you're importing all of datetime.

you really only need these two, I think.
```
from datetime import datetime, timedelta 
```
should suffice. Creates less bloat